scripts/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -136,8 +136,10 @@ def submit_eqtl_jobs(
                 )
 
                 keys = os.path.join(input_path, 'OneK1K_CPG_IDs.tsv')
+                version = output_dir.split('/')[-1]
+                # put version number at end of output path
                 output_prefix = os.path.join(
-                    output_dir, f'{cell_type}', f'chr{chromosome}'
+                    output_dir.strip(version), f'{cell_type}', f'chr{chromosome}', f'{version}'
```

Comment:
`strip` is probably not what you want here, but `removesuffix`. (See https://stackabuse.com/python-remove-the-prefix-and-suffix-from-a-string/.)

Reply:
neat, I didn't know about `removesuffix`